### DISCUSSION TITLE
[Feature] [Mobile] "Upload with Immich"

### DISCUSSION THREAD

**Discussion by Aseeef** [immich-app/immich#14423] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When you select media on your phone in an app outside Immich (say the "Gallery" for example), you are able to share the image on supported apps.

My suggestion is to add an option such that when you select "share", you are able to "Upload with Immich". The App That Shall Not Be Named has this feature and its quite useful especially for bulk uploading media selected in another app to immich.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#2659
